readers/postgres/messages.go

```diff
@@ -122,11 +123,54 @@ func fmtCondition(chanID string, query map[string]string) string {
 			"name",
 			"protocol":
 			condition = fmt.Sprintf(`%s AND %s = :%s`, condition, name, name)
+		case "v":
+			condition = fmt.Sprintf(`%s AND value = :value`, condition)
+		case "vb":
+			condition = fmt.Sprintf(`%s AND bool_value = :bool_value`, condition)
+		case "vs":
+			condition = fmt.Sprintf(`%s AND string_value = :string_value`, condition)
+		case "vd":
+			condition = fmt.Sprintf(`%s AND data_value = :data_value`, condition)
+		case "from":
+			condition = fmt.Sprintf(`%s AND time >= :from`, condition)
+		case "to":
+			condition = fmt.Sprintf(`%s AND time < :to`, condition)
 		}
 	}
 	return condition
 }
 
+func fmtCount(chanID string, query map[string]string) (string, []interface{}) {
```

Comment:
Can named variables be used here (similar to the previous case)? I think that `sqlx` does not allow it, but please double-check. If it can be done, these two methods can be merged, because they will mostly be the same.